workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```diff
@@ -360,12 +360,12 @@ describe('WBCGroupManagemntPlugin', () => {
       ).rejects.toThrow(PluginConfigurationError);
     });
 
-    it('throws GroupNotFoundError when the group doesnt exist', async () => {
-      mockUserManagementPlugin.listUsersForRole = jest.fn().mockRejectedValue(new GroupNotFoundError());
+    it('throws RoleNotFoundError when the Role doesnt exist', async () => {
```

Comment:
I probably wasnt clear, user management throws a `RoleNotFoundError`, wbc group management catches that error and turns it into a `GroupNotFoundError`